extensions/authentication-jwt/src/types.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/authentication-jwt
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {UserProfile} from '@loopback/security';
+
+export type RefreshGrant = {
+  refreshToken: string;
+};
+
+export const RefreshGrantSchema = {
+  type: 'object',
+  required: ['refreshToken'],
+  properties: {
+    refreshToken: {
+      type: 'string',
+    },
+  },
+};
+export const RefreshGrantRequestBody = {
+  description: 'Reissuing Acess Token',
+  required: true,
+  content: {
+    'application/json': {schema: RefreshGrantSchema},
+  },
+};
+
+export type TokenObject = {
+  accessToken: string;
+  expiresIn?: string | undefined;
+  refreshToken?: string | undefined;
+};
+
+export interface RefreshTokenService {
+  /**
+   * Generate Token and return the Token Object
+   */
+  generateToken(userProfile: UserProfile, token: string): Promise<TokenObject>;
+  /**
+   * Verifies the validity of a token string and returns a new Token
+   *
+   */
+  refreshToken(refreshToken: string): Promise<TokenObject>;
+}
```

Comment:
Nitpick: I would like to see descriptive tsdocs for exported types/interfaces/constants.